lib/watir/window.rb

```diff
@@ -190,6 +196,9 @@ def url
     #
 
     def use(&blk)
+      if @browser
```

Comment:
same as above. The plan is to remove this conditional & the deprecation notice whenever we do Watir 7